pages/api/applications/[id]/evidence.tsx

```diff
@@ -0,0 +1,31 @@
+import { StatusCodes } from 'http-status-codes';
+import type { NextApiHandler, NextApiRequest, NextApiResponse } from 'next';
+import { createEvidenceRequest } from '../../../../lib/gateways/applications-api';
+
+const endpoint: NextApiHandler = async (
+  req: NextApiRequest,
+  res: NextApiResponse
+) => {
+  switch (req.method) {
+    case 'POST':
+      try {
```

Comment:
There doesn't seem to be any access checking going on here or elsewhere? Does that mean that anyone could set up evidence requests for an application?

Reply:
read the cookie
check that the JWT application id  matches the [id] in the url